ccpp/CMakeLists.txt

```diff
@@ -54,14 +50,6 @@ if(MPI)
   add_definitions(-DMPI)
 endif()
 
-#------------------------------------------------------------------------------
-# Set additional flags for debug build
-if(DEBUG)
-  if(CMAKE_Fortran_COMPILER_ID MATCHES "Intel")
-    set(CMAKE_Fortran_FLAGS "${CMAKE_Fortran_FLAGS} -init=snan,arrays")
```

Comment:
Just want to confirm, we do not need this for DEBUG mode any more?

Reply:
No, this is now the default in ufs-weather-model/cmake/Intel.cmake: https://github.com/ufs-community/ufs-weather-model/blob/9b6b7402de2c06eaf121c4a9a4285b651a936c5d/cmake/Intel.cmake#L10

It wasn't there until a while ago.